Dockerfile

```diff
@@ -25,15 +24,17 @@ WORKDIR /tmp
 
 #Install Ruby
 COPY .ruby-version .ruby-version
-RUN curl "https://cache.ruby-lang.org/pub/ruby/ruby-$(cat .ruby-version).tar.gz" | \
-  tar -xz && \
-  cd ruby-$(cat .ruby-version) && \
-  ./configure --enable-shared --disable-install-doc > /dev/null && \
-  make -s -j4 >> install_ruby.log && make -s install > /dev/null && \
-  rm /usr/local/lib/libruby-static.a
+
+#Download RVM as root
+RUN \curl -#LO https://rvm.io/mpapis.asc && gpg --import mpapis.asc && \
+  \curl -sSL https://get.rvm.io | bash -s stable
```

Comment:
Get the following trace when running `./script/docker_compose` - is this all normal?
```bash
Step 15/24 : RUN \curl -#LO https://rvm.io/mpapis.asc && gpg --import mpapis.asc &&   \curl -sSL https://get.rvm.io | bash -s stable
 ---> Running in 56e2f71d8a18
######################################################################## 100.0%
gpg: directory `/root/.gnupg' created
gpg: new configuration file `/root/.gnupg/gpg.conf' created
gpg: WARNING: options in `/root/.gnupg/gpg.conf' are not yet active during this run
gpg: keyring `/root/.gnupg/secring.gpg' created
gpg: keyring `/root/.gnupg/pubring.gpg' created
gpg: /root/.gnupg/trustdb.gpg: trustdb created
gpg: key D39DC0E3: public key "Michal Papis (RVM signing) <mpapis@gmail.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1  (RSA: 1)
gpg: no ultimately trusted keys found
Downloading https://github.com/rvm/rvm/archive/1.29.3.tar.gz
Downloading https://github.com/rvm/rvm/releases/download/1.29.3/1.29.3.tar.gz.asc
gpg: Signature made Sun Sep 10 20:59:21 2017 UTC using RSA key ID BF04FF17
gpg: Good signature from "Michal Papis (RVM signing) <mpapis@gmail.com>"
gpg:                 aka "Michal Papis <michal.papis@toptal.com>"
gpg:                 aka "[jpeg image of size 5015]"
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 409B 6B17 96C2 7546 2A17  0311 3804 BB82 D39D C0E3
     Subkey fingerprint: 62C9 E5F4 DA30 0D94 AC36  166B E206 C29F BF04 FF17
GPG verified '/usr/local/rvm/archives/rvm-1.29.3.tgz'
Creating group 'rvm'

Installing RVM to /usr/local/rvm/
Installation of RVM in /usr/local/rvm/ is almost complete:

  * First you need to add all users that will be using rvm to 'rvm' group,
    and logout - login again, anyone using rvm will be operating with `umask u=rwx,g=rwx,o=rx`.

  * To start using RVM you need to run `source /etc/profile.d/rvm.sh`
    in all your open shell windows, in rare cases you need to reopen all shell windows.
Removing intermediate container 56e2f71d8a18
 ---> f2f3555ca52d
```

Reply:
This is normal really and to be expected, we can suppress the log output which we can consider going forward. However this should only occur on the initial build of the image.